apps/web/ofn_web.gemspec

```diff
@@ -0,0 +1,17 @@
+$:.push File.expand_path("../lib", __FILE__)
+
+# require "ofn_web/version"
+
+Gem::Specification.new do |s|
+  s.name        = "ofn_web"
+  s.version     = "0.0.1"
+  s.authors     = ["ofn"]
+  s.email       = ["hello@openfoodnetwork.org"]
+  s.homepage    = "http://www.openfoodnetwork.org/"
+  s.summary     = "OFN frontoffice web app"
+  s.description = "Consumer facing OFN frontoffoce web application"
+
+  s.files = Dir["{app,config,lib}/**/*"] + ["README.rdoc"]
+
+  s.add_dependency "rails", "~> 3.2.22.5"
+end
```

Comment:
the gem should be named after its folder. Besides, I don't see the point about the redundancy of the `ofn_` prefix. This is OFN's repo so I don't there's need for it; it can be annoying to have a bunch of engines all of them with this prefix.

Reply:
done 👍